hubgrep/lib/hosting_service_interfaces/get_hosting_service_interfaces.py

```diff
@@ -53,21 +49,13 @@ def get_hosting_service_interfaces():
     for service in app.config["CACHED_HOSTING_SERVICES"]:
         service: HostingService
 
-        config_str = service.config
-        config = json.loads(config_str)
-
-        SearchClass = hosting_service_interfaces_by_name[service.type]
         cache_backend = _get_cache_backend()
         logger.debug(f"using cache backend: {cache_backend}")
```

Comment:
we got rid of the unfolded config here, and moved the actual hosting_service_interface creation to the model